.travis.yml

```diff
@@ -32,16 +33,19 @@ install:
   - if [ ! -z $CHECK_CLA ] && [ "$TRAVIS_PULL_REQUEST" != "false" ]; then $DEPENDENCIES; fi
   - if [ ! -z $TEST_SUITE ]; then docker run --name builder -e TEST_USER_EMAIL=$TEST_USER_EMAIL
     -e TEST_USER_PASSWORD=$TEST_USER_PASSWORD -e TEST_STORE=$TEST_STORE
-    -e TEST_SNAP_WITH_TRACKS=$TEST_SNAP_WITH_TRACKS -e LC_ALL=en_US.UTF-8
+    -e TEST_SNAP_WITH_TRACKS=$TEST_SNAP_WITH_TRACKS
+    -e GITHUB_TEST_USER_NAME=$GITHUB_TEST_USER_NAME -e GITHUB_TEST_PASSWORD=$GITHUB_TEST_PASSWORD
+    -e LC_ALL=en_US.UTF-8
     -v $PWD:$PWD -w $PWD -td ubuntu:xenial; fi
   - if [ ! -z $TEST_SUITE ]; then docker exec -i builder locale-gen en_US.UTF-8; fi
   - if [ ! -z $TEST_SUITE ]; then docker exec -i builder sed -i s/archive.ubuntu.com/us.archive.ubuntu.com/g /etc/apt/sources.list; fi
   - if [ ! -z $TEST_SUITE ]; then docker exec -i builder apt update; fi
   - if [ ! -z $TEST_SUITE ]; then docker exec -i builder sh -c "$DEPENDENCIES"; fi
 
 script:
-  - if [ ! -z $CHECK_CLA ] && [ "$TRAVIS_PULL_REQUEST" != "false" ]; then ./tools/cla_check_in_travis.sh; fi
-  - if [ ! -z $TEST_SUITE ]; then docker exec -i builder ./runtests.sh $TEST_SUITE; fi
+  - if [ ! -z $CHECK_CLA ] && [ "$TRAVIS_PULL_REQUEST" != "false" ] && [ "$TRAVIS_EVENT_TYPE" != 'cron' ]; then ./tools/cla_check_in_travis.sh; fi
```

Comment:
No, because it would run when the branch is landed in master. That's != pull request and != cron.